postman-collection/runner/tests.postman_collection.json

```diff
@@ -0,0 +1,800 @@
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "{{access_token}}",
+											"type": "string"
+										}
+									]
+								},
+								"method": "GET",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "\"result\"",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{url}}/api/internal/v1/hearings/{{hearing_id}}",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"api",
+										"internal",
+										"v1",
+										"hearings",
+										"{{hearing_id}}"
+									]
+								}
+							},
+							"response": []
+						}
+					]
+				},
+				{
+					"name": "V2",
+					"item": [
+						{
+							"name": "Hearing results",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"hearing object is present in the response\", function () {",
+											"    var json_data = pm.response.json();",
+											"    pm.expect(json_data.hearing).to.exist;",
+											"    pm.expect(json_data.hearing).to.be.a('object');",
+											"});",
+											"",
+											"",
+											"pm.test(\"court_centre is present in the response\", function () {",
+											"    var json_data = pm.response.json();",
+											"    pm.expect(json_data.hearing.court_centre).to.exist;",
+											"    pm.expect(json_data.hearing.court_centre).to.be.a('object');",
+											"});",
+											"",
+											"pm.test(\"prosecution_cases is present in the response\", function () {",
+											"    var json_data = pm.response.json();",
+											"    pm.expect(json_data.hearing.prosecution_cases).to.exist;",
+											"    pm.expect(json_data.hearing.prosecution_cases).to.be.a('array');",
+											"});"
+										],
+										"type": "text/javascript",
+										"packages": {}
+									}
+								}
+							],
+							"protocolProfileBehavior": {
+								"disableBodyPruning": true
+							},
+							"request": {
+								"auth": {
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "{{access_token}}",
+											"type": "string"
+										}
+									]
+								},
+								"method": "GET",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "",
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{url}}/api/internal/v2/hearing_results/{{hearing_id}}",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"api",
+										"internal",
+										"v2",
+										"hearing_results",
+										"{{hearing_id}}"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Hearing event log",
```

Comment:
Nit: We're including v1 hearing results but not hearing event log. Shouldn't we be consistent and do both?